File: SecondYear/Parth27904_ParthAgarwal_2226CSE(AI&ML)1015_2/Week_2/fibonacci.java
public class fibonacci {
    public static int[] generateFibonacciNumbers(int n) {
        // Write your code here.
        int[] array = new int[n];
        array[0] = 0;
        if(n==1){
            return array;
        }
        array[1]= 1;
        for(int i = 2; i<n; i++){
            array[i] = array[i-1] + array[i-2];
        }
        return array;
    }
}

File: SecondYear/Akash-Tiwari22_AkashTiwari_2226cs1058_2/Week_2/fibonacci.cpp
vector<int> generateFibonacciNumbers(int n) {
    // Write your code here.
    vector<int>fibonacci;
    if(n<1)
    return fibonacci;

fibonacci.push_back(0);
if (n == 1) {
    return fibonacci;
}
fibonacci.push_back(1);
if (n == 2) {
    return fibonacci;
}

for (int i = 2; i < n; i++) {
    int ans = fibonacci[i - 1] + fibonacci[i - 2];
    fibonacci.push_back(ans);
}
return fibonacci;
}